.pre-commit-config.yaml

```diff
@@ -6,20 +6,23 @@ repos:
     hooks:
       - id: trailing-whitespace
       - id: end-of-file-fixer
-        exclude: ^up42/_version\.txt$
+        exclude: ^up42/_version\.txt$|.github/CODEOWNERS
       - id: check-added-large-files
         args: ["--maxkb=1000"]
+      - id: check-yaml
+      - id: mixed-line-ending
+
   - repo: https://github.com/psf/black
     rev: 23.3.0
     hooks:
       - id: black
         args: ['--line-length', '120']
-        exclude: ^.*__init__\.py$|^tests/
+        exclude: ^.*__init__\.py$|^tests/|.github/CODEOWNERS
```

Comment:
This very strange, all this exclusions are not needed

Reply:
They are required currently due to an error in github governance repo. We will tackle in a future ticket.